src/containers/tutor-home-page/general-info-step/GeneralInfoStep.tsx

```diff
@@ -20,23 +20,17 @@ import { type UserGeneralInfo, type UserRole } from '~/types'
 
 interface GeneralInfoStepProps {
   btnsBox: ReactNode
-  isUserFetched: boolean
-  setIsUserFetched: (isUserFetched: boolean) => void
 }
 
 type UserName = { firstName: string; lastName: string }
 
-const GeneralInfoStep = ({
-  btnsBox,
-  isUserFetched,
-  setIsUserFetched
-}: GeneralInfoStepProps) => {
+const GeneralInfoStep: React.FC<GeneralInfoStepProps> = ({ btnsBox }) => {
   const { t } = useTranslation()
   const { isLaptopAndAbove, isMobile } = useBreakpoints()
   const { stepData, handleGeneralInfo } = useStepContext()
   const { userId, userRole } = useAppSelector((state) => state.appMain)
   const generalInfo = stepData.generalInfo
-
+  const firstRender = useRef<boolean>(true)
```

Comment:
Let's name it `isFirstRender` please
```suggestion
  const isFirstRender = useRef<boolean>(true)
  
```